{"errorMessage": "An error occurred (UnauthorizedOperation) when calling the DescribeInstances operation: You are not authorized to perform this operation. User: arn:aws:sts::390403862276:assumed-role/lambda_inventory_role/ResourceInventoryLambda is not authorized to perform: ec2:DescribeInstances because no identity-based policy allows the ec2:DescribeInstances action", "errorType": "ClientError", "requestId": "a7bef601-3f9f-4a1f-81ac-d77720a625d6", "stackTrace": ["  File \"/var/task/lambda_function.py\", line 19, in lambda_handler\n    instances = ec2.describe_instances()\n", "  File \"/var/runtime/botocore/client.py\", line 569, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/runtime/botocore/client.py\", line 1023, in _make_api_call\n    raise error_class(parsed_response, operation_name)\n"]}